src/pages/plan/itinerary/PlanItineraryPage.tsx

```
@@ -791,3 +832,101 @@ export function PlanItineraryPage() {
+  goBack: () => void
+  handleNext: () => void
+  changeDay: (day: number) => void
+  handleRecommendQueryChange: (value: string) => void
+  handleCancelDeparture: () => void
+}) {
+  const goBackRef = useRef(goBack)
+  const handleNextRef = useRef(handleNext)
+  const changeDayRef = useRef(changeDay)
+  const searchRef = useRef(handleRecommendQueryChange)
+  const cancelRef = useRef(handleCancelDeparture)
+  goBackRef.current = goBack
```

ESLint

[error] Error: Cannot access refs during render

React refs are values that are not needed for rendering. Refs should only be accessed outside of render, such as in event handlers or effects. Accessing a ref value (the `current` property) during render can cause your component not to update as expected (https://react.dev/reference/react/useRef).

/home/jailuser/git/src/pages/plan/itinerary/PlanItineraryPage.tsx:868:3
  866 |   const searchRef = useRef(handleRecommendQueryChange)
  867 |   const cancelRef = useRef(handleCancelDeparture)
> 868 |   goBackRef.current = goBack
      |   ^^^^^^^^^^^^^^^^^ Cannot update ref during render
  869 |   handleNextRef.current = handleNext
  870 |   changeDayRef.current = changeDay
  871 |   searchRef.current = handleRecommendQueryChange

(react-hooks/refs)